nntrainer/tensor/tensor_base.h

```diff
@@ -21,6 +21,34 @@
 #include <tensor_dim.h>
 #include <util_func.h>
 
+#define transposeloop(cl, ci, cj, ck, sl, si, sj, sk)                 \
```

Comment:
I'm not sure if it is working with omp directives in define. If it can, then we could try to apply it.

Reply:
agree. I think we should test it after when openMP is enabled. I'll keep it in mind and come back to this later.